products/jbrowse-cli/src/base.ts

```diff
@@ -49,19 +49,6 @@ export default abstract class JBrowseCommand extends Command {
   }
 
   async readJsonConfig(location: string) {
-    let locationUrl: URL | undefined
```

Comment:
I believe readJsonConfig should not be hitting the network with fetch? This seems unexpected